### ISSUE TITLE
Ultrawidescreen (32:9) HUD and FOV issue

### ISSUE THREAD

**Issue by darkdrifter** [multitheftauto/mtasa-blue#2753]
### Describe the bug

Camera FOV seems to be based off screen width instead of screen height, thus cropping view on the top and bottom when using a wider screen instead of extending the view on the sides.
HUD is also based off width to control the scaling, instead so instead of "extending" the HUD on the left and right of the screen while preserving the scaling of the height, it gets twice as big on a 32:9 compared to a 16:9.
Single player with ThirteenAG's wide screen fix doesn't have this issue. 
Using the FOV slider doesn't improve the situation, as it is clamped to 90° (horizontal), so to have a the same 59° of vertical FOV (for a 90° horizontal FOV on 16:9), it would need to be set at 127° (horizontal) on a 32:9.

Comparison:
![GTA SA solo ultrawidescreen](https://user-images.githubusercontent.com/7112422/190900556-32ed0609-ad5a-4aa1-98c6-df2fd77eb90a.jpg)
![MTA SA ultrawidescreen](https://user-images.githubusercontent.com/7112422/190900560-13941497-b261-4c6c-9a0e-f8bb3f63075c.jpg)

### Steps to reproduce

1. Launch the game
2. Set the video resolution to a 32:9 aspect
3. Reboot the game to apply the resolution change
4. Launch a server or map editor
5. Observe the low FOV and big HUD.

### Version

Client: version 1.5.9-release-21324

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by Haxardous** [comment]
#848

**Comment by darkdrifter** [comment]
> #848

Yes I saw this issue, mentions HUD scaling but not camera FOV, so it's not a duplicate.

**Comment by Grzehu71** [comment]
Hello, any update in this matter? Since ASI loaders are not allowed - the widescreen fix doesn't work on MTA.
The "auto" aspect ratio is not scaling HUD properly - so issue persists till this day. 
Could you add at least manual option to choose 32:9 or 32:10 aspect ratio?
